app/docs/dev/policies/meshaccesslog.md

```diff
@@ -0,0 +1,293 @@
+```
+
+Example configuration:
+
+```yaml
+format:
+  plain: '[%START_TIME%] %BYTES_RECEIVED%'
+```
+
+Example output:
+
+```text
+[2016-04-15T20:17:00.310Z] 154
+```
+
+#### JSON
+
+Example configuration:
+
+```yaml
+format:
+  json:
+    - key: "start_time"
+      value: "%START_TIME%"
+    - key: "bytes_received"
+      value: "%BYTES_RECEIVED%"
+```
+
+Example output:
+
+```json
+{
+  "start_time": "2016-04-15T20:17:00.310Z",
+  "bytes_received": "154"
+}
+```
+
+<details>
+  <summary>TCP configuration with default fields:</summary>
+
+```yaml
+format:
+  json:
+    - key: "start_time"
+      value: "%START_TIME%"
+    - key: "response_flags"
+      value: "%RESPONSE_FLAGS%"
+    - key: "kuma_mesh"
+      value: "%KUMA_MESH%"
+    - key: "kuma_source_address_without_port"
+      value: "%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%"
+    - key: "kuma_source_service"
+      value: "%KUMA_SOURCE_SERVICE%"
+    - key: "upstream_host"
+      value: "%UPSTREAM_HOST%"
+    - key: "kuma_destination_service"
+      value: "%KUMA_DESTINATION_SERVICE%"
+    - key: "duration_ms"
+      value: "%DURATION%"
+    - key: "bytes_sent"
+      value: "%BYTES_SENT%"
+    - key: "bytes_received"
+      value: "%BYTES_RECEIVED%"
+```
+
+</details>
+
+<details>
+  <summary>HTTP configuration with default fields:</summary>
+
+```yaml
+format:
+  json:
+    - key: "start_time"
+      value: "%START_TIME%"
+    - key: "kuma_mesh"
+      value: "%KUMA_MESH%"
+    - key: 'method'
+      value: '"%REQ(:METHOD)%'
+    - key: "path"
+      value: "%REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%"
+    - key: 'protocol'
+      value: '%PROTOCOL%'
+    - key: "response_code"
+      value: "%RESPONSE_CODE%"
+    - key: "response_flags"
+      value: "%RESPONSE_FLAGS%"
+    - key: "bytes_received"
+      value: "%BYTES_RECEIVED%"
+    - key: "bytes_sent"
+      value: "%BYTES_SENT%"
+    - key: "duration_ms"
+      value: "%DURATION%"
+    - key: "upstream_service_time"
+      value: "%RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%"
+    - key: 'x_forwarded_for'
+      value: '"%REQ(X-FORWARDED-FOR)%"'
+    - key: 'user_agent'
+      value: '"%REQ(USER-AGENT)%"'
+    - key: 'request_id'
+      value: '"%REQ(X-REQUEST-ID)%"'
+    - key: 'authority'
+      value: '"%REQ(:AUTHORITY)%"'
+    - key: "kuma_source_service"
+      value: "%KUMA_SOURCE_SERVICE%"
+    - key: "kuma_destination_service"
+      value: "%KUMA_DESTINATION_SERVICE%"
+    - key: "kuma_source_address_without_port"
+      value: "%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%"
+    - key: "upstream_host"
+      value: "%UPSTREAM_HOST%"
+```
+
+</details>
+
+
+### Backends
+
+#### TCP
+
+You can configure a `TCP` backend with an address:
+
+```yaml
+backends:
+  - tcp:
+      address: 127.0.0.1:5000
+```
+
+#### File
+
+You can configure a `File` backend with a path:
```

Comment:
```suggestion
You can configure a file backend with a path:
```

Can we be deliberate with our formatting here? Why "a `File` backend"? `File` makes me think of a resource called File.